lib/plugins/create/templates/aws-clojure-gradle/serverless.yml

```diff
@@ -27,6 +27,9 @@ provider:
   timeout: 300
   memorySize: 1536
   logRetentionInDays: 30
+  lambdaHashingVersion: 20201221
+  apiGateway:
+    shouldStartNameWithService: true
```

Comment:
I think we may leave this out of this update. 

Template by default doesn't configure `http` event, so not adding that doesn't leave us with deprecation thrown upfront.

What we may do though is update example events below to promote `httpApi` instead of `http` (while `httpApi` is still a bit more limited, is way less problematic)

Reply:
I was a bit torn on this one, as we're offering "commented out" `http` event in these templates, but I agree with the argument. Do you suggest to change the examples from `http` to `httpApi` as a part of this PR as well?

Reply:
> Do you suggest to change the examples from http to httpApi as a part of this PR as well?

I think `httpApi` should be favored over `http`, it resembles cleaner API, and on CloudFormation configuration side doesn't share issues as we have with API Gateway v1. What's your opinion?

Changing examples I think would be simply about renaming event (as basic config is same)

Reply:
Sounds good to me, I'll update it :+1: